test/benchmark/lmbench-ctx/config.json

```diff
@@ -1,5 +1,7 @@
 {
     "alert_threshold": "125%",
     "pattern": "18 ",
-    "field": "2"
+    "field": "2",
+    "tool": "customSmallerIsBetter",
```

Comment:
Is this field name chosen by you, all the original Github action? This name is too vague in my opinion.

In fact, all the key names of this configuration json, except `alert_threshold`, are not self-describing. It is unclear what the "pattern" is for, what the "field" refers to, and what the "tool" is used for.

Reply:
Only `tool` and `alert_threshold` are from the action, but we can still rename them. I think we can rename `pattern` for `search_pattern`, `field` for `target_field` and `tool` for `validate_tool`.